pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/QueryRunner.java

```diff
@@ -118,6 +122,7 @@ public void init(PinotConfiguration config, InstanceDataManager instanceDataMana
           getQueryWorkerIntermExecutorService());
       _leafScheduler = new OpChainSchedulerService(new RoundRobinScheduler(releaseMs), getQueryRunnerExecutorService());
       _mailboxService = new MailboxService(_hostname, _port, config, _intermScheduler::onDataAvailable);
+      _exchangeService = new ExchangeService(_hostname, _port, config);
```

Comment:
i am not sure this should be a separate service or should we put together with mailbox. b/c ultimately this is submitting a thread to poll data out of the exchange queue and sending data via GRPC or in-mem mailboxes.

Reply:
discussed offline. removing and put the change back into mailbox service